addons/Automatic_Feedback/src/presenter.js

```diff
@@ -472,19 +525,51 @@ function AddonAutomatic_Feedback_create() {
         lastClass = "";
 
         onCorrectAnswer(itemID) {
-            this.onShowErrorsMode();
+            var _class = FEEDBACK_CLASSES.CORRECT;
+            var feedback = "";

```

Comment:
proponuję utworzyć osobną metodę zwracającą feedback:
getFeedback(itemID, feedbackType) {
            if (itemID && presenter.configuration.itemFeedbacks[itemID]) {
               return presenter.configuration.itemFeedbacks[itemID][feedbackType];
            }
            var feedbackObject = this.presenter.configuration.defaultFeedback;
            return feedbackObject[feedbackType];
}

dodając funkcję powyżej można by rozważyć dodanie zmiannej np. 
var FEEDBACK_TYPE = {
          CORRECT: "correct",
          INCORRECT: "incorrect",
          PARTIAL: "partial",
          EMPTY: "empty"
}
zastępujące wypisywanie typu tutaj jak i w konstrukcji switch/case z linii 440